File: roteiro_03/cpp_logic/desafio4.cpp
#include <iostream>
using namespace std;

int main() {
    int numero_secreto = 7;
    int palpite = 0;

    cout << "--- Jogo de Adivinhacao ---" << endl;


    while (palpite != numero_secreto) {
        cout << "Digite seu palpite: ";
        cin >> palpite;


        if (palpite > numero_secreto) {
            cout << "Tente um numero menor!" << endl;
        } else if (palpite < numero_secreto) {
            cout << "Tente um numero maior!" << endl;
        }
    }

   
    cout << "\nParabens! Voce acertou o numero secreto!" << endl;

    return 0;
}

File: roteiro_03/java_logic/desafios/desafio4.java
package roteiro_03.java_logic.desafios;

import java.util.Scanner;

public class desafio4 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int numsecret = 10;
        int palpite = 0;

        System.out.println("----- JOGO DE ADIVINHAÇÃO-----");
        System.out.println("Tente adivinhar o numero secreto!");

        while(palpite!= numsecret){
            System.out.print("Digite seu palpite:");
            palpite = sc.nextInt();
            if (palpite > numsecret){
                System.out.println("TEnte um numero menor!");
            }else if (palpite < numsecret){
                System.out.println("Tente um numeor maior!");
            }
        }
    
        System.out.println("Parabens! voce acertou o numero secreto.");
        sc.close();  
    }
    
}
